securedrop_client/gui/widgets.py

```diff
@@ -895,6 +944,43 @@ def trigger(self):
             self.messagebox.launch()
 
 
+class JournalistMenu(QMenu):
+    """A menu next to the journalist username.
+
+    A menu that provides login options.
+    """
+
+    def __init__(self, parent):
+        super().__init__()
+        self.logout = QAction(_('SIGN OUT'))
+        self.logout.setFont(QFont("Helvetica [Cronyx]", 10))
+        self.addAction(self.logout)
+        self.logout.triggered.connect(parent.on_logout_clicked)
+
+
+class JournalistMenuButton(QToolButton):
+    """An menu button for the journalist menu
+
+    This button is responsible for launching the journalist menu on click.
+    """
+
+    def __init__(self, parent):
+        super().__init__()
+
+        self.setStyleSheet('''
+            QToolButton::menu-indicator { image: none; }
+            QToolButton { border: none; }
+        ''')
+        arrow = load_image("dropdown_arrow.svg")
+        self.setIcon(QIcon(arrow))
+        self.setMinimumSize(20, 20)
```

Comment:
is it me or do these pixmaps look kind of blocky? (just for the dropdown SVG and the refresh SVG)

<img width="131" alt="Screen Shot 2019-03-21 at 4 42 30 PM" src="https://user-images.githubusercontent.com/7832803/54791735-5b76dd80-4bf8-11e9-8ead-8ac49c08c171.png">

note that it's possible this is a macOS only thing - let me know if so and we can disregard since we're targeting Linux.

Reply:
Blarg—is that crappy art on my part, @creviera? If so, happy to re-rip another one... 🤢

Reply:
@ninavizz - Some higher fidelity images would be great! I'm currently using the svgs provided in the inventory list.

Reply:
@ninavizz and @redshiftzero - In a follow-up PR I was able to improve the look of the refresh icon by changing this (which looked pixely/blocky):

```py
        refresh_pixmap = load_image('refresh.svg')
        self.refresh.setIcon(QIcon(refresh_pixmap))
```

To the following code:
```py
        pixmap = load_image('refresh.svg')
        scaled = pixmap.scaled(18, 18, Qt.KeepAspectRatio, transformMode=Qt.SmoothTransformation)
        icon.addPixmap(scaled, QIcon.Normal)
```

But I think I need to do more research into how to work with SVGs and QIcons. I may need to create a `QSvgWidget` which are used to display the contents of SVG files: https://doc.qt.io/archives/qt-5.10/qtsvg-module.html